doc/whats_new.rst

```diff
@@ -231,7 +231,7 @@ Enhancements
      By `Sebastian Säger`_ and `YenChen Lin`_.
 
    - Added new return type ``(data, target)`` : tuple option to
-     :func:`load_iris` dataset.
+     :func:`load_iris` dataset, :func:`load_breast_cancer` dataset
      (`#7049 <https://github.com/scikit-learn/scikit-learn/pull/7049>`_) by
```

Comment:
im inclined to have it formatted as such:
Add new return option to load (link to pr), loader2 (link to another pr), etc.